www/src/styles/global.css

```diff
@@ -159,26 +159,29 @@
   }
 
   .markdown .remark-code-title {
-    @apply z-10 -mb-1 mt-3 w-full rounded-t-md border-b-2 border-t3-purple-200 bg-[#292D3E] px-2 py-1 font-mono text-t3-purple-200;
+    @apply z-10 -mb-1 mt-3 w-full rounded-t-md border border-t3-purple-300/80 bg-[#191724] px-2 py-1 font-mono text-t3-purple-200;
   }
 
   .markdown pre {
-    @apply mx-auto mb-3 rounded-md p-2 pl-3 font-mono sm:pt-2;
+    @apply mx-auto mb-3 rounded border border-t3-purple-300/80 p-2 pl-3 font-mono shadow-xl sm:pt-2;
   }
 
   .markdown ol {
     @apply w-full;
   }
 
   .markdown details {
-    @apply mt-2 w-full rounded-md  bg-[#292D3E] px-3 transition-all duration-75 ease-in-out;
+    @apply mt-2 w-full rounded-md bg-[#191724] border border-t3-purple-300/80 px-3 transition-all duration-75 ease-in-out;
```

Comment:
```suggestion
    @apply mt-2 w-full rounded-md bg-[#191724] border border-t3-purple-300/80 px-3 transition-all duration-300 ease-in-out;
```

Reply:
should the quote maybe be a component? makes it easier to keep them consistent, especially if we were to add more of them in the future

Reply:
probably